packages/amplify_datastore_plugin_interface/test/testData/PostAuthComplex.dart

```diff
@@ -64,14 +64,14 @@ class PostAuthComplex extends Model {
     buffer.write("PostAuthComplex {");
     buffer.write("id=" + id + ", ");
     buffer.write("title=" + title + ", ");
-    buffer.write("owner=" + owner);
+    buffer.write("owner=" + owner!);
```

Comment:
Is this a correct casting? Can't `owner` still be `null` here?

Reply:
Yes.  For now I am ignoring the Dart codegen models as that is a bit of a larger discussion that we can discuss in the PR for Null Safety upgrade of Datastore.